src/components/auth/signup/DisplayMessage.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const DisplayMessage = props => props.message && (
```

Comment:
Hello Marcus, nice job here, but you could destructure 'message' and 'onSubmit' from the props, or what do you think?